tests/k8s/1-032_validate_dynamic_scaling/02-assert.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd
+status:
+  phase: Available
+---
+apiVersion: v1
+kind: Pod
```

Comment:
Rather than the pod, can we assert on the `StatefulSet` object's `.status.availableReplicas` value ? Current style of assertion does not guarentee that there are exactly 3 pods. Even if there are 5 pods, this test would still succeed.